src/ballet/sbpf/fd_opcode.h

```diff
@@ -0,0 +1,226 @@
+#ifndef HEADER_fd_src_ballet_sbpf_fd_opcode_h
+#define HEADER_fd_src_ballet_sbpf_fd_opcode_h
+
+/* This header defines the opcodes used in the sBPF (Solana BPF)
+   instruction set. The opcode is the first byte of each instruction
+   slot. */
+
+/* Opcode encoding helpers ********************************************/
+
+/* BPF_OPC: Opcode number given the operation class, mode, sub-operation. */
+#define BPF_OPC( class, mode, variant ) (uchar)( (BPF_CLASS_##class)|(BPF_##mode)|(BPF_##variant) )
```

Comment:
Names and prefixes better but still not fully harmonized (this file suggests the prefixes are SBPF but the prefixes are BPF below).  I'm partial to three letter prefix than a four for readability / speakability and partial for disambiguating this from Berkeley Packet Filter as it is evolving (e.g. SBC for Solana Byte Code, SIS for Solana Instruction Set, etc).

Also, the rationale for not having a FD_ prefix on public exposed macros / symbols should be documented at the top.